ee/frontend/mobile-replay/schema/web/rr-web-schema.json

```diff
@@ -631,7 +658,7 @@
                     "type": "number"
                 }
             },
-            "required": ["id", "source", "type", "x", "y"],
+            "required": ["id", "source", "type"],
```

Comment:
x and y aren't required anymore?

Reply:
Exactly, comes from this change that was merged: https://github.com/rrweb-io/rrweb/pull/1375